cli/tests/script_url.ts

```diff
@@ -0,0 +1,3 @@
+console.log("script_url", Deno.scriptUrl.pathname);
```

Comment:
What's the intuition behind the name `scriptUrl`? How about `mainUrl`?

Reply:
🤷‍♂️  I started out with `scriptPath` before realising Deno can also execute remote files directly, so I replaced the `-Path` suffix with `-Url`. Didn't dwell on the nomenclature any further than that.

Will fix.